pkg/minikube/registry/drvs/docker/docker.go

```diff
@@ -102,7 +102,7 @@ func status() (retState registry.State) {
 	ctx, cancel := context.WithTimeout(context.Background(), 6*time.Second)
 	defer cancel()
 
-	cmd := exec.CommandContext(ctx, oci.Docker, "version", "--format", "{{.Server.Os}}-{{.Server.Version}}")
+	cmd := exec.CommandContext(ctx, oci.Docker, "version", "--format", "{{.Server.Os}}-{{.Server.Version}}:{{.Server.Platform.Name}}")
```

Comment:
will this fail on Docker Linux without DD ?

Reply:
No, it returns `Docker Engine - Community` for `{{.Server.Platform.Name}}`